hopr/hopr-lib/src/lib.rs

```diff
@@ -689,44 +736,30 @@ impl Hopr {
         self.state.store(HoprState::Indexing, Ordering::Relaxed);
 
         let (to_process_tx, to_process_rx) =
-            async_channel::bounded::<IndexerToProcess>(core_transport::constants::INDEXER_UPDATE_QUEUE_SIZE);
+            async_channel::bounded::<IndexerTransportEvent>(core_transport::constants::INDEXER_UPDATE_QUEUE_SIZE);
```

Comment:
What is the relevance of the queue size? Should it be configurable?

Reply:
Good question. Ideally, only the ingress receivers should be limited, if at all. For the indexer, we should never be able to fill up the queue and we definitely do not want the discard.

Reply:
Removed.